geonode/documents/api/views.py

```diff
@@ -137,7 +137,7 @@ def perform_create(self, serializer):
 
             resource.set_missing_info()
             resourcebase_post_save(resource.get_real_instance())
-            resource_manager.set_permissions(None, instance=resource, permissions=None, created=True)
+            resource.set_default_permissions(owner=self.request.user)
```

Comment:
in this case we should keep `created=True` since we are in the creating workflow